packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -373,63 +380,84 @@ function TokensPanel({
     }
   }
 
+  function onSearchInputChange(event: React.ChangeEvent<HTMLInputElement>) {
+    setErrorMessage('')
+    setNewToken(event.target.value)
+  }
+
+  const rowRenderer = useCallback(
+    (virtualizedProps: ListRowProps) => {
+      const address = tokensToShow[virtualizedProps.index]
+      let token: ERC20BridgeToken | null = null
+
+      if (isTokenArbitrumOneNativeUSDC(address)) {
+        token = ARB_ONE_NATIVE_USDC_TOKEN
+      } else if (isTokenArbitrumSepoliaNativeUSDC(address)) {
+        token = ARB_SEPOLIA_NATIVE_USDC_TOKEN
+      } else if (address) {
+        token = tokensFromLists[address] || tokensFromUser[address] || null
+      }
+
+      if (address === NATIVE_CURRENCY_IDENTIFIER) {
+        return (
+          <TokenRow
+            key="TokenRowNativeCurrency"
+            onClick={() => onTokenSelected(null)}
```

Comment:
(same for L415), we should not pass anonymous function

Reply:
can't do `useCallback` in a `useCallback` (rowRenderer)

Reply:
You can pass `onClick={onTokenSelected}` directly, and from `TokenRow` call `onTokenSelected(token)`